LICENSE.md

```diff
@@ -1,9 +1,9 @@
-<!-- SPDX-FileCopyrightText: Copyright 2021 Arm Limited and/or its affiliates <open-source-office@arm.com> -->
+<!-- SPDX-FileCopyrightText: Copyright 2021, 2023 Arm Limited and/or its affiliates <open-source-office@arm.com> -->
 <!-- SPDX-License-Identifier: Apache-2.0 -->
 
 # License for the specifications
 
-All the ACLE documents themselves are not dependent on any assets
+All the current ACLE documents themselves are not dependent on any assets
```

Comment:
When I read `current` without any other context I wonder what current means exactly. When is a document current.

Reply:
Fixed. "current" -> "open-source", and made it clearer that the legacy ones are closed-source.